data/kernels/bilateral.cl

```diff
@@ -281,8 +281,8 @@ blur_line(
 kernel void
 slice_to_output(
     read_only  image2d_t in,
+    read_only  image2d_t output,
```

Comment:
maybe "target" is a suitable name to not get into too much logical conflict with the kernel's name ("slice_to_output").